src/index.js

```diff
@@ -676,6 +698,24 @@ function dataLoaded(geomapFeatures, allDates, baseData) {
     const slider = resetSlider(allDates);
     resetGeoMap(geomapFeatures);
 
+    const top5 = svg.append("g")
+        .attr("class", "top5");
+
+    var xValTop5 = 40;
+    var yValTop5 = 360;
+    top5.append("text").attr("class", "top5Text").attr("x", xValTop5).attr("y", yValTop5 - 20)
+    for (let i = 0; i < 5; i++) {
+        top5.append("text").attr("class", "text"+yValTop5).attr("x", xValTop5).attr("y", yValTop5);
+        yValTop5 = yValTop5 + 20;
+    }
+
+    const title = svg.append("g")
+        .attr("class", "title");
```

Comment:
[nit] There might be other scenarios that would conflict with this class so it's better to use something like "top5Title"

Reply:
But if you're not referencing this class you don't need to set it as an attribute